iModelCore/GeomLibs/geom/src/polyface/IPolyfaceConstruction_Add.cpp

```diff
@@ -2553,6 +2573,8 @@ bool IPolyfaceConstruction::AddRuledBetweenCorrespondingCurves
                 }
             if (capped)
                 context.EmitCaps (reversedPrimary);
+            // Bug #1279805: disconnects used to separate loops get copied into mesh xyz and prevent future clipping

```

Comment:
I dont think we want to merge bug numbers in our repos!

Reply:
I'm not aware of any proscription on this, but I agree that it doesn't add value these days. Old habits die hard!